ci_tools/check_pylint_commands.py

```diff
@@ -68,15 +108,15 @@ def check_expected_pylint_disable(file, disabled, flag, messages):
 
     success = True
 
-    messages = []
+    messages = {"output":[]}
```

Comment:
You don't need to save the dictionary `{"output":[]}` to file, just the value. The outer dictionary which contains similar information for each check run will be built elsewhere.
Also beware the "output" field is an object as per the docs:
https://docs.github.com/en/rest/checks/runs?apiVersion=2022-11-28#create-a-check-run
Therefore it should be a dictionary not a list. Only the "annotations" which have type `array of objects` should be a list